### PR TITLE
Update gsd_prep_chem to GEFS v12.3.2

### PR DESCRIPTION
**Pull request by lipan-NOAA**
Updates the prep chem version for GEFS v12.3

### CODE REVIEW

**Comment by lipan-NOAA** [comment]
@XianwuXue-NOAA @WalterKolczynski-NOAA please review it

File: sorc/checkout.sh [hunk 1]
```diff
@@ -87,7 +87,7 @@ if [[ ! -d gsd_prep_chem.fd ]] ; then
     rc=$?
     ((err+=$rc))
     cd gsd_prep_chem.fd
-    git checkout gefs_v12.2.0
+    git checkout gefs_v12.3.2
```
Comment (XianwuXue-NOAA) [inline sorc/checkout.sh]: I think the tag Sam created is [gefs.v12.3.2](https://github.com/NOAA-GSL/GSL-prep-chem/releases/tag/gefs.v12.3.2). Please double check

**Review by XianwuXue-NOAA — CHANGES_REQUESTED** [review]

File: sorc/checkout.sh (hunk 1 above)
Comment (lipan-NOAA) [reply sorc/checkout.sh]: you are right.

**Review by XianwuXue-NOAA — APPROVED** [review]

**Comment by XianwuXue-NOAA** [comment]
@WalterKolczynski-NOAA Could you please review the PR and create new tag: gefs_v12.3.2? Thanks a lot in advance.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Comment by XianwuXue-NOAA** [comment]
@WalterKolczynski-NOAA Thanks for your approval, when you are available, could you create a new tag: gefs_v12.3.2?

**Pull request merged**